gammapy/irf/io.py

```diff
@@ -19,6 +13,45 @@
 }
 
 
+# The key is the class tag. The hdu name is not strictly declared in the GADF
+# format
+# TODO: extend the info here
+IRF_DL3_HDU_SPECIFICATION = {
+    "bkg_3d": {
+        "hdu": "BACKGROUND",
```

Comment:
Here it is `extname`  rather than `hdu` no?

Reply:
Done